app/components/form/TeamContacts.tsx

```diff
@@ -105,6 +126,32 @@ export default function TeamContacts({
             </div>
           ) : null}
         </div>
+        <div className="mt-6 flex flex-col flex-start">
+          <h3 className="text-base 2xl:text-xl font-semibold leading-7 text-gray-900">
+            After-Hours Support Contact (optional)
+          </h3>
+          <p className="my-4 text-base leading-6 text-gray-600">
+            For Business Mission Critical Applications Only. You can specify a phone number of your team member who
+            should be contacted by the BC Government&apos;s Service Desk (7-7000) if an issue is reported for your
+            product outside of business hours
+          </p>
+          <IMaskInput
+            className="w-fit"
+            mask="+1 (000) 000-0000"
+            unmask={true}
+            value={phoneNumber}
+            onAccept={handlePhoneNumberChange}
+            placeholder="+1 (999) 999-9999"
+          />
+          <p
+            className={classNames(
+              errors.supportPhoneNumber ? 'text-red-400' : '',
```

Comment:
I'm wondering if `text-red-400` will override `text-gray-600` even though it is defined before `text-gray-600`.